source/contracts/reporting/ReportingWindow.sol

```diff
@@ -283,6 +318,9 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
             reporterStatus[_reporter].finishedReporting = true;
         }
         numberOfReportsByMarket[_market] += 1;
+        // Record gas price
+        reportingGasPriceSum = reportingGasPriceSum.add(tx.gasprice);
+        numReports++;
```

Comment:
Since these two variables are inextricably linked, I wonder if there would be readability value in creating a library for `RunningAverage` like:
```
library RunningAverage {
    struct Data {
        uint256 sum;
        uint256 count;
    }
    function currentAverage(Data self) internal returns (uint256) {
        return self.sum / self.count;
    }
    function record(Data self, uint256 input) internal returns (bool) {
        self.count++;
        self.sum += input;
        return bool;
    }
}
```
I'm not sure if we'll use this construct elsewhere or not, but I find it nice to be able to keep these two pieces of data together in a construct that makes it clear to the reader how they are related.

Reply:
Side-note: If both of these can be fit into `uint128` we can save on storage costs.

Reply:
Actually, it appears we already have two constructs like this putting up a minor argument for DRY.

Reply:
Seems like a good idea, will do!